akka-remote/src/main/scala/akka/remote/transport/netty/NettyTransport.scala

```diff
@@ -340,16 +340,16 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
 
   private val associationListenerPromise: Promise[AssociationEventListener] = Promise()
 
-  private def sslHandler(isClient: Boolean): SslHandler = {
-    val handler = NettySSLSupport(settings.SslSettings.get, log, isClient)
+  private def sslHandler(remoteAddress: Option[Address], isClient: Boolean): SslHandler = {
```

Comment:
perhaps we should split this in a `sslServerHandler` and `sslClientHandler` to make explicit that a server handler never has a `remoteAddress` and a client always does?